code/modules/surgery/organs/organ_external.dm

```diff
@@ -552,6 +523,51 @@ Note that amputating the affected organ does in fact remove the infection from t
 			qdel(src) // If you flashed away to ashes, YOU FLASHED AWAY TO ASHES
 			return null
 
+/obj/item/organ/external/chest/droplimb()
```

Comment:
The big problem you have here is this is tied to the chest.

This needs to be its own proc, then both chests and groins need to override `droplimb` and call this proc.

We want to be able to spill the organs in a groin as well as a chest, since they're separate here, unlike on TG.

Reply:
okay coding this tipsy watch me get it wrong..